dts/bindings/display/fsl,imx6sx-lcdif.yaml

```
@@ -14,5 +14,94 @@ properties:
     interrupts:
       required: true
 
-    label:
+    hsync:
+      type: int
       required: true
+      description: HSYNC pulse width
+
+    hfp:
+      type: int
+      required: true
+      description: Horizontal front porch
+
+    hbp:
+      type: int
+      required: true
+      description: Horizontal back porch
+
+    vsync:
+      type: int
+      required: true
+      description: VSYNC pulse width
+
+    vfp:
+      type: int
+      required: true
+      description: Vertical front porch
+
+    vbp:
+      type: int
+      required: true
+      description: Vertical back porch
+
+    polarity:
+      type: int
+      enum:
+        - 0
+        - 1
+        - 2
+        - 3
+        - 4
+        - 5
+        - 6
+        - 7
+        - 8
+        - 9
+        - 10
+        - 11
+        - 12
+        - 13
+        - 14
+        - 15
+      description:
+        OR'ed value of elcdif_polarity_flags, used to control the signal polarity.
+        0000 VSYNC active low, HSYNC active low, Drive data on falling edge, Data enable active low.
+        0001 VSYNC active low, HSYNC active low, Drive data on falling edge, Data enable active high.
+        0010 VSYNC active low, HSYNC active low, Drive data on rising edge, Data enable active low.
+        0011 VSYNC active low, HSYNC active low, Drive data on rising edge, Data enable active high.
+        0100 VSYNC active low, HSYNC active high, Drive data on falling edge, Data enable active low.
+        0101 VSYNC active low, HSYNC active high, Drive data on falling edge, Data enable active high.
+        0110 VSYNC active low, HSYNC active high, Drive data on rising edge, Data enable active low.
+        0111 VSYNC active low, HSYNC active high, Drive data on rising edge, Data enable active high.
+        1000 VSYNC active high, HSYNC active low, Drive data on falling edge, Data enable active low.
+        1001 VSYNC active high, HSYNC active low, Drive data on falling edge, Data enable active high.
+        1010 VSYNC active high, HSYNC active low, Drive data on rising edge, Data enable active low.
+        1011 VSYNC active high, HSYNC active low, Drive data on rising edge, Data enable active high.
+        1100 VSYNC active high, HSYNC active high, Drive data on falling edge, Data enable active low.
+        1101 VSYNC active high, HSYNC active high, Drive data on falling edge, Data enable active high.
+        1110 VSYNC active high, HSYNC active high, Drive data on rising edge, Data enable active low.
+        1111 VSYNC active high, HSYNC active high, Drive data on rising edge, Data enable active high.
+
+    pixel-format:
+      type: int
+      enum:
+        - 0
+        - 1
+      description:
+        Display pixel format
+        0 RGB_888.
+        1 BGR_565.
```

```suggestion
      type: int
      enum:
        - "rgb-888"
        - "bgr-565"
      description:
        Display pixel format
```

and use DT_ENUM_IDX()

---

Thanks for the suggestion, I have made the change.